README.md

```diff
@@ -166,8 +166,8 @@ DEFAULT pxeboot
 TIMEOUT 20
 PROMPT 0
 LABEL pxeboot
-    KERNEL fedora-coreos-30.20191014.1-live-kernel-x86_64
-    APPEND ip=dhcp rd.neednet=1 initrd=fedora-coreos-30.20191014.1-live-initramfs.x86_64.img console=tty0 console=ttyS0 coreos.inst.install_dev=/dev/sda coreos.inst.stream=testing coreos.inst.ignition_url=http://192.168.1.101:8000/config.ign
```

Comment:
This change looks good to me. I have one question: If someone was running the install via an Ignition config and they added `ignition.firstboot ignition.config.url`, the conditional networking stuff will appropriately detect `ignition.config.url` and bring up networking, right?

Reply:
Yup.  The Ignition `fetch-offline` stage performs a normal config acquisition using a disabled fetcher, and then asks for network if that fetcher is ever run.